### PR TITLE
Fix Akamai signature generation

### PR DESCRIPTION
**Pull request by dmitry-mukhin**
## Checklist

- [ ] Tests (if applicable)
- [ ] Documentation (if applicable)
- [ ] Changelog stub (or use [conventional commit messages](https://www.conventionalcommits.org/))

### CODE REVIEW

**Review by ivanredbread — APPROVED** [review]
I have seen parallel libs for such URL construction - this one seems to be good!

File: pyuploadcare/secure_url.py [hunk 1]
```diff
@@ -87,7 +88,7 @@ def _build_signature(self, expire: int, acl: str) -> str:
         ]
 
         signature = hmac.new(
-            self.secret_key.encode(),
+            binascii.a2b_hex(self.secret_key.encode()),
```
Comment (homm) [inline pyuploadcare/secure_url.py:91]: Can't find any links to reference or docs.
No info here https://uploadcare.com/docs/security/secure_delivery/

Comment (homm) [reply pyuploadcare/secure_url.py:91]: I want documentation how to get signature. Where I can read that `a2b_hex` is required here?

Comment (dmitry-mukhin) [reply pyuploadcare/secure_url.py:91]: the secret key should be a hexadecimal string:  https://techdocs.akamai.com/property-mgr/docs/auth-token-2-0-ver
`hmac.new` accepts bytes as a key, hence `a2b_hex`.

in [the reference](https://github.com/akamai/EdgeAuth-Token-Python/blob/master/akamai/edgeauth/edgeauth.py#L175-L179) Akamai code they are doing a similar thing

**Review by homm — APPROVED** [review]

**Pull request merged**